composer.json

```diff
@@ -376,7 +370,8 @@
             "web/libraries/{$name}": [
                 "type:drupal-library",
                 "type:bower-asset",
-                "type:npm-asset"
+                "type:npm-asset",
+                "enyo/dropzone"
```

Comment:
I think this line can be removed as well since that module is no longer installed.

Reply:
Removed